README.md

```diff
@@ -146,15 +146,18 @@ Some ingredients that can enhance your test suite are:
 * [tasty-stats](https://hackage.haskell.org/package/tasty-stats) adds the
   possibility to collect statistics of the test suite in a CSV file.
 
+### Test discovery
+
+`tasty` by itself forces you to explicitly write out the `TestTree` yourself. The packages listed below allow you to write tests at the top-level, and will automatically collect them into a single `TestTree`.
```

Comment:
```suggestion
`tasty` by itself forces you to explicitly write out the `TestTree` yourself. The packages listed below
allow you to write tests at the top-level, and will automatically collect them into a single `TestTree`.
```

Nit: the entire README has somewhat shorter lines; if you don't mind keeping it that way would be nice. I won't die on this hill though.

Reply:
Yeah, I can do that. IMO markdown files should always keep everything on one line and let editors wrap, because manually wrapping lines means you have to rewrap lines when you edit lines, and can make formatting weird on narrow screens (e.g. mobile). But being consistent with the rest of the README is more important